src/applications/edu-benefits/components/EducationWizard.jsx

```diff
@@ -76,7 +76,6 @@ class EducationWizard extends React.Component {
       <a
         id="apply-now-link"
         href={url}
-        className="usa-button va-button-primary"
```

Comment:
When clicked, the behavior of this link changes the URL. This means that it should not be presented as a button so I have removed that style and kept it as an anchor tag.

Reference [Links vs Buttons](https://design.va.gov/components/button/#links-vs-buttons):

> A good rule is if the action changes the URL, it should not be a button.
Button and link confusion can be very frustrating for assistive technology users. 

> A user with a screen reader may pull up a list of links and may not find a specific link because it turns out that it’s been designated as a button in the markup.

Reply:
I also want to mention that I'm not replacing it with `va-link` because the click callback `recordWizardValues` is sending a custom analytics event and the va-link web component is not customizable in that way.